### ISSUE TITLE
Improve UI/UX of the website

### ISSUE THREAD

**Issue by gauravsinhaweb** [gauravsinhaweb/EduSmart#7]
update the color palette, update new SVGs, make the website UI/UX better.

**Comment by parth-panchal** [comment]
Hi @gauravsinhaweb, could you assign this issue to me, please? I would love to work on this.

**Comment by gauravsinhaweb** [comment]
@parth-panchal sure, go for it.

**Comment by TishaJhabak1014** [comment]
@gauravsinhaweb can you assign me an issue to add a new section of faqs?

**Comment by gauravsinhaweb** [comment]
@TishaJhabak1014 Yes, you can do that. Create an issue in https://github.com/gauravsinhaweb/EduSmart/issues and start working on it. Let me know if you need any help :)

**Comment by TishaJhabak1014** [comment]
> @TishaJhabak1014 Yes, you can do that. Create an issue in https://github.com/gauravsinhaweb/EduSmart/issues and start working on it. Let me know if you need any help :)

Issue created. Plz assign me and add the labels. Thank you.

**Comment by rajputarjun2001** [comment]
Hi @gauravsinhaweb , I want to work on this could you please assign this to me?

**Comment by gauravsinhaweb** [comment]
Hi @rajputarjun2001 , Go for it 👍

**Comment by gauravsinhaweb** [comment]
Hi, @parth-panchal , let me know if you're still working in it.

**Comment by parth-panchal** [comment]
> Hi, @parth-panchal , let me know if you're still working in it.

Hi. Due to other engagements I couldn't find the time to work on this, that's why I unassigned myself from the issue, so that others who want to work can be assigned. So currently I am not working on it.

**Comment by gauravsinhaweb** [comment]
> > Hi, @parth-panchal , let me know if you're still working in it.
> 
> Hi. Due to other engagements I couldn't find the time to work on this, that's why I unassigned myself from the issue, so that others who want to work can be assigned. So currently I am not working on it.

Thanks for letting me know about it. Feel free to PR anytime you want.

**Comment by SupriyaKumari08** [comment]
Hey @gauravsinhaweb ,if this issue is still active...I would like to work on it

**Comment by gauravsinhaweb** [comment]
> Hey @gauravsinhaweb ,if this issue is still active...I would like to work on it

 assigned to you
